Source/WebCore/workers/service/ServiceWorkerGlobalScope.idl

```diff
@@ -38,6 +38,7 @@
 
     [NewObject] Promise<undefined> skipWaiting();
 
+    attribute EventHandler oncookiechange;
```

Comment:
Is this the correct way to deal with:

``
```suggestion
    partial interface [ServiceWorkerGlobalScope] {
         [SameObject] readonly attribute CookieStore cookieStore;
         
         attribute EventHandler oncookiechange    
```

Because there is a build failure and I suspect it has to do with this?

Reply:
I'm still looking into if this is what's causing the build failure:

<img width="1686" alt="image" src="https://github.com/WebKit/WebKit/assets/42721308/d5d64e72-acf4-4ce8-ae8c-e30d00908350">

Once I figure out the failure, I should be able to rebasline new tests that hopefully pass. But I still wanted to upload the pr as a draft to get a look at it in the meantime.

Reply:
Ok I've managed to fix this issue. 'cookie change' had to be added to EventNames.h. I am working on getting layout tests to pass and will update the pr then.